<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>653</width>
        <height>784</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="toolTip">
          <string>Weight rasters generated in Calculate weights step. Generalized W+ and W+ rasters should be used.</string>
         </property>
         <property name="title">
          <string>Weight rasters</string>
         </property>
         <layout class="QVBoxLayout" name="weight_rasters_layout"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="toolTip">
          <string>Standard deviation rasters generated in Calculate weights step. Generalized S_W+ and S_W+ rasters should be used.</string>
         </property>
         <property name="title">
          <string>Standard deviation rasters</string>
         </property>
         <layout class="QVBoxLayout" name="standard_deviation_rasters_layout"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="weights_table_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="title">
          <string>Weights table</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QgsMapLayerComboBox" name="weights_table">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Weight metrics table generated in Calculate weights step. Needed for total and deposit pixel counts.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="toolTip">
             <string>Raster of posterior probabilites.</string>
            </property>
            <property name="text">
             <string>Posterior probabilities</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsFileWidget" name="output_probabilities">
            <property name="toolTip">
             <string>Raster of posterior probabilites.</string>
            </property>
            <property name="storageMode">
             <enum>QgsFileWidget::SaveFile</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>Raster of standard deviations in the posterior probability calculations.</string>
            </property>
            <property name="text">
             <string>Posterior probabilities std</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="toolTip">
             <string>Raster of confidence of the prospectivity values obtained in the posterior probability array.</string>
            </property>
            <property name="text">
             <string>Posterior confidence</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsFileWidget" name="output_probabilities_std">
            <property name="toolTip">
             <string>Raster of standard deviations in the posterior probability calculations.</string>
            </property>
            <property name="storageMode">
             <enum>QgsFileWidget::SaveFile</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QgsFileWidget" name="output_confidence">
            <property name="toolTip">
             <string>Raster of confidence of the prospectivity values obtained in the posterior probability array.</string>
            </property>
            <property name="storageMode">
             <enum>QgsFileWidget::SaveFile</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QTextEdit" name="log">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="autoFormatting">
            <set>QTextEdit::AutoNone</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="progress_layout">
           <item>
            <widget class="QLabel" name="progress_label">
             <property name="text">
              <string>Progress</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progress_bar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
